frontend/javascripts/oxalis/view/right-border-tabs/skeleton_tab_view.js

```diff
@@ -219,14 +221,33 @@ export async function importTracingFiles(files: Array<File>, createGroupForEachF
           const dataFile = new File([dataBlob], dataFileName);
 
           await Model.ensureSavedState();
-          const { tracing, dataset } = Store.getState();
-          const oldVolumeTracing = tracing.volume;
+          const storeState = Store.getState();
+          const { tracing, dataset } = storeState;
+          if (tracing.volumes.length === 0) {
+            throw new Error("A volume tracing must already exist when importing a volume tracing.");
+          }
+          const oldVolumeTracing = getActiveSegmentationTracing(storeState);
+          if (oldVolumeTracing == null) {
+            throw new Error(
+              "Ensure that a volume tracing layer is visible when importing a volume tracing.",
```

Comment:
In theory, this could be extended now by asking the user whether the volume tracing should be merged to the active volume tracing layer or be imported as a separate volume tracing layer. Maybe we can add this to a followup issue.

Reply:
Good point. I added it to https://github.com/scalableminds/webknossos/issues/5695.